styles.css

```
@@ -1,420 +1,450 @@
-}
+.save-load-btn.secondary { color: #27332f; background: #b8b38f; }
+.save-load-btn:hover { background: #a93429; transform: translateY(-2px); box-shadow: 0 6px 0 #08100e; }
+.save-load-btn:active { transform: translateY(3px); box-shadow: 0 1px 0 #08100e; }
+.save-load-btn:focus-visible, .modal-btn:focus-visible, .traindex-link:focus-visible { outline: 3px solid var(--signal); outline-offset: 3px; }
 
-#btn-up { grid-column: 2; grid-row: 1; }
-#btn-left { grid-column: 1; grid-row: 2; }
-#btn-center { grid-column: 2; grid-row: 2; opacity: 0; pointer-events: none; }
-#btn-right { grid-column: 3; grid-row: 2; }
-#btn-down { grid-column: 2; grid-row: 3; }
+.button-code { display: grid; place-items: center; height: 27px; color: var(--brass-light); border: 1px solid currentColor; font-size: 7px; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)